collector/perf_linux.go

```diff
@@ -726,9 +720,6 @@ func (c *perfCollector) updateCacheStats(ch chan<- prometheus.Metric) error {
 		if err := (*profiler).Profile(cacheProfile); err != nil {
 			return err
 		}
-		if cacheProfile == nil {
-			continue
-		}
```

Comment:
Hmm, I'm not sure about this one. What is the warning for this?

Reply:
I will post all warnings, but cacheProfile warning was similar with hwProfile, swProfile.

collector/helper.go:51:1: comment on exported function SanitizeMetricName should be of the form "SanitizeMetricName ..." (ST1020)
collector/lnstat_linux.go:52:10: error strings should not be capitalized (ST1005)
collector/perf_linux.go:53:16: error strings should not be capitalized (ST1005)
collector/perf_linux.go:601:6: this nil check is never true (SA4031)
	collector/perf_linux.go:597:17: this is the value of hwProfile
collector/perf_linux.go:673:6: this nil check is never true (SA4031)
	collector/perf_linux.go:669:17: this is the value of swProfile
collector/perf_linux.go:729:6: this nil check is never true (SA4031)
	collector/perf_linux.go:725:20: this is the value of cacheProfile